src/utils/articleVideo/articlePlayerOnReady.ts

```diff
@@ -0,0 +1,38 @@
+import { communicationService } from 'utils/communication';
+import { setVideoSeenInSession } from 'utils/articleVideo/articleVideoSession';
+import { willAutoplay, willMute } from 'utils/articleVideo/articleVideoConfig';
+
+export default function useOnArticlePlayerReady(videoDetails, playerInstance): void {
+	const playerKey = 'aeJWPlayerKey';
```

Comment:
Is there any significance to the 'ae' prefix?

Reply:
I think it stands for ad engine. The ad engine code checks for that one specifically .